mips2c_macros.h

```diff
@@ -0,0 +1,17 @@
+#ifndef _MIPS2C_MACROS_H_
+#define _MIPS2C_MACROS_H_
+
+// Unknown types
+#define MIPS2C_UNK      s32
+#define MIPS2C_UNK8     s8
+#define MIPS2C_UNK16    s16
+#define MIPS2C_UNK32    s32
+#define MIPS2C_UNK64    s64
+
+// Bitwise (reinterpret) cast
+#define MIPS2C_BITWISE(_type, _expr) (*(_type *)&(_expr))
```

Comment:
why the underscores?

note that this won't compile if expr is an rvalue, so another option, bad in another way, is to define this as the identity function

Reply:
Underscores is just a habit from a different style, you're right they're not needed here. 

And I originally thought it was rare for expr to not be an rvalue - but it's more common than I thought.  I changed this to just a plain cast.